events/events-team/content/meet-and-greet.md

```diff
@@ -0,0 +1,94 @@
+
+# Follow-ups
+
+Check weekly to see who's signed up.  While participation by all SIGs is valuable, the following SIGs are particularly popular with 1st-time contributors, and if nobody signs up you should message them:  SIG-Docs, SIG-Release, SIG-CLI, SIG-Contributor Experience, SIG-K8s-Infra.
+
+### 1 Week Before: Solicit Attendees and Send Reminders
+
+By this time, CNCF events should have given you a room number and day. Send out a reminder to all of the chairs and techleads, and give them the updated information including the location and time.  Update the Summit website with the full information.
+
+Working with Comms, advertise the Meet & Greet to Kubecon attendees and community members.  This should include advertising it on community channels, including social media, #contributor-summit, #kubernetes-contributors, and kubernetes-dev ML, and LWKD.  You should also make sure that it gets into at least one of Kubecon's official messages to attendees; ask CNCF about this.
+
+Stretch Goal: prepare a post-M&G survey.  We haven't actually done one of these before, so you'll be starting from scratch.  Good questions to ask: how do they feel it went, how many folks from their SIG were there, and how many contributors did they talk to (and maybe a newbie/experienced estimate).  For attendees, it would ask which SIG(s) they were seeking, why, and if they found what they were looking for. Print QR codes for the survey to have on the tables.
+
+Decide whether you'll have a sign-in sheet for attendees.  This is unnecessary if you're doing a survey, but if you're not, you should at least have sign-ins. If so, design & print it out.
+
+### 1 Day Before
+
+If possible, check out the space assigned to the M&G, just in case there are problems with it.  If there are, raise them with our CNCF event staff.
+
+### Day Of
+
+Get to the room 15-30 minutes before SIG volunteers are due to arrive.  Work with CNCF staff to distribute the SIG packs among the tables.  SIGs will have to share tables, so try to combine them in ways that make sense, such as SIG-Testing with SIG-K8s-Infra and SIG-Networking with SIG-Multicluster.  Consult the signup sheet, though, to see who's actually expected to show up.  While signups are not reliable, they will tell you which SIGs are likely to have large contingents so that you don't seat them together.
+
+If you don't have shadow(s), then recruit a couple folks from SIG-ContribEx to help with setup.
+
+Each pack has SIG buttons in it, and a table stand with a sign.  Set up the table stands for the SIGs who RSVPd; leave the others in the bag.
+
+The SIG-Beard pack is special. Leave it in the box unless someone who knows what to do with it shows up.
+
+Wear some kind of Kubernetes gear, to help identify you as staff.
+
+### During the M&G
+
+Hover near the entrance to help attendees find the SIGs they are looking for.   Sometimes they won't know, so be ready to chat about their interests with them and direct them appropriately.
```

Comment:
We also coordinated with comms to send out pictures on social, and sometimes to share pictures on how to get where you need to go.

Reply:
Suggest language?